cmd/kubeadm/app/cmd/init.go

```diff
@@ -277,14 +277,21 @@ func (i *Init) Run(out io.Writer) error {
 
 	adminKubeConfigPath := filepath.Join(kubeConfigDir, kubeadmconstants.AdminKubeConfigFileName)
 
-	// PHASE 1: Generate certificates
-	if err := certsphase.CreatePKIAssets(i.cfg); err != nil {
-		return err
-	}
+	if res, err := certsphase.UsingExternalCA(i.cfg); !res {
+		fmt.Printf("[externalca] Not using external CA mode: %v\n", err)
```

Comment:
I don't think we should print anything here, this is the "normal" state anyway...

Reply:
Removed the print, if someone wants to see why external CA mode is not active they would have to add print statements to this step.  (a logger with verbosity levels would be nice but it seems to be not used in kubeadm)